pallets/common/src/traits/balances.rs

```diff
@@ -123,6 +126,13 @@ decl_event!(
         /// The account and the amount of unlocked balance of that account that was burned.
         /// (caller Id, caller account, amount)
         AccountBalanceBurned(IdentityId, AccountId, Balance),
+        /// Some balance was reserved (moved from free to reserved).
+        Reserved(AccountId, Balance),
```

Comment:
I guess we could add Option<IdentityId> as the first arg here, but if it is easier you can just capture this as a new story in the jira backlog.